### PR TITLE
fix(core/webidl): fully qualified names not coded

### LINKED ISSUE #1437: Fully qualified method names are not coded
The draft [Web Share](https://wicg.github.io/web-share/) spec has recently (not sure when?) broken all the links to dictionary members. This previously worked.

I have:

```html
      <section data-dfn-for="ShareData">
        <pre class="idl">
          dictionary ShareData {
            USVString title;
            USVString text;
            USVString url;
          };
        </pre>
        ... snip ...
        <dfn>title</dfn>: The title of the document being shared.
```

And later in the doc:

```html
    <a for="ShareData">title</a>
```

This used to properly link to ... I don't remember. It either linked to the "title" member defined in the IDL, or the `<dfn>title</dfn>` definition element within the ShareData section. I don't remember exactly how it worked but it used to work.

Now all of my `<a for="ShareData">title</a>` and similar refs are just broken refs. Did anything change that may have accidentally broken this?

### PR DESCRIPTION
**Pull request by marcoscaceres**
* closes #1437

### CODE REVIEW

File: src/core/webidl.js [hunk 1]
```diff
@@ -612,17 +612,37 @@ function findDfn(parent, name, definitionMap, type, idlElem) {
...
-      const asFullyQualifiedName = parent + "." + name.toLowerCase() + "()";
+      const asLocalName = name.toLowerCase();
+      const asMethodName = asLocalName + "()";
+      const asQualifiedName = parent + "." + asLocalName;
+      const asFullyQualifiedName = asQualifiedName + "()";
 
-      if (definitionMap[asMethodName] || definitionMap[asFullyQualifiedName]) {
+      if (
+        definitionMap[asMethodName] ||
+        definitionMap[asFullyQualifiedName.toLowerCase()]
```
Comment (sidvishnoi) [inline src/core/webidl.js]: Are we sure we need a `.toLowerCase()` here?

Comment (marcoscaceres) [reply src/core/webidl.js]: yeah, definitely. everything in the definition map is lowercase.

Comment (marcoscaceres) [reply src/core/webidl.js]: (eventually we should have a idlDefinitionMap, which is case sensitive)

Comment (sidvishnoi) [reply src/core/webidl.js]: Oh yes. I missed we haven't done it yet.

**Comment by marcoscaceres** [comment]
Thanks for the speedy review!

**Pull request merged**